source/userguide/augmentation.rst

```diff
@@ -0,0 +1,103 @@
+Data Augmentation in MindMeld
+=============================
+
+MindMeld provides the ability to augment data to collect more diverse queries and make the applications more robust.
```

Comment:
When starting out to build a new conversational application in a custom domain, it is often the case that we have very limited training data to train models. In such scenarios, one way of bootstrapping your models is by using data augmentation techniques to increase the size of our training set. MindMeld now provides the ability to augment data via automatic paraphrasing in English and other languages to help make the applications more robust.